source/application/BioXAS/BioXASSideAppController.cpp

```diff
@@ -45,6 +59,10 @@ bool BioXASSideAppController::setupDataFolder()
 void BioXASSideAppController::initializeBeamline()
 {
 	BioXASSideBeamline::bioXAS();
+	connect( BioXASSideBeamline::bioXAS()->ge32ElementDetector(), SIGNAL(connected(bool)), this, SLOT(onGeDetectorConnectedChanged()) );
+	onGeDetectorConnectedChanged();
+
+	setupScanConfigurations();
```

Comment:
`setupScanConfigurations()` is called in `CLSAppController::startup()`, you don't need to call it here.

If you think this should happen earlier, you might want to change it in `CLSAppController::startup()`